spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/neo4j/Neo4jHealthContributorConfigurations.java

```diff
@@ -38,7 +38,10 @@
  * @author Michael J. Simons
  * @author Stephane Nicoll
  */
-class Neo4jHealthContributorConfigurations {
+final class Neo4jHealthContributorConfigurations {
```

Comment:
I did this change for a lot of "configurations" classes which are no configurations themself, but are `@Import`ed.